lib/Test/Script.pm

```diff
@@ -7,10 +7,11 @@ package Test::Script;
 
  use Test2::V0;
  use Test::Script;
- 
+
```

Comment:
I'm good :+1: with most of the white space fixes, however, (and I realize that this is somewhat controversial), my preference is to keep this one character indent of white space inside of a literal block for blank lines, because some POD renderers (probably incorrectly) otherwise render this as two separate literal blocks, which annoys me.

Reply:
Done